common: fix spelling errors

ldorau opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/pmem/nvml/1128)
<!-- Reviewable:end -->

krzycz commented:
Reviewed 41 of 41 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[utils/docker/README, line 10 \[r1\]](https://reviewable.io:443/reviews/pmem/nvml/1128#-KQVE-E8Pm-GbRRTslow:-KQVE-E8Pm-GbRRTslox:b9fjcf4) ([raw file](https://github.com/pmem/nvml/blob/06fe5aeb66a74c72a001d2952385ae3062e6d846/utils/docker/README#L10)):*
> ```
> NOTE:
> (for those, who have Travis builds enabled for their fork of the NVML project)
> If you commit changes to any Dockerfile or shell script in the 'images'
> ```

I think it was intentional. If changing, use "Docker image" or "Dockerfile.*"

---

*[src/test/unittest/unittest.h, line 124 \[r1\]](https://reviewable.io:443/reviews/pmem/nvml/1128#-KQVCqRmpVeNGnypFore:-KQVCqRnFcqVGohFLpVy:b-3qovov) ([raw file](https://github.com/pmem/nvml/blob/06fe5aeb66a74c72a001d2952385ae3062e6d846/src/test/unittest/unittest.h#L124)):*
> ```objc
> #define UT_POOL_HDR_UUID_GEN_FILE "/proc/sys/kernel/random/uuid"
> 
> /* XXX - fix this temp hack dup'ing util_strerror when we get mock for win */
> ```

I'm not sure if this should be changed.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/nvml/1128)*
<!-- Sent from Reviewable.io -->

ldorau commented:
Review status: 37 of 38 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/test/unittest/unittest.h, line 124 \[r1\]](https://reviewable.io:443/reviews/pmem/nvml/1128#-KQVCqRmpVeNGnypFore:-KQVbWbA-X5MLr6imAaH:bwvyox4) ([raw file](https://github.com/pmem/nvml/blob/06fe5aeb66a74c72a001d2952385ae3062e6d846/src/test/unittest/unittest.h#L124)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary>
> I'm not sure if this should be changed.
</details>
Reverted

---

*[utils/docker/README, line 10 \[r1\]](https://reviewable.io:443/reviews/pmem/nvml/1128#-KQVE-E8Pm-GbRRTslow:-KQVbcux6cFBG0zC_ATb:bwvyox4) ([raw file](https://github.com/pmem/nvml/blob/06fe5aeb66a74c72a001d2952385ae3062e6d846/utils/docker/README#L10)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary>
> I think it was intentional. If changing, use "Docker image" or "Dockerfile.*"
</details>
Reverted

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/nvml/1128)*
<!-- Sent from Reviewable.io -->

krzycz commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 3 of 4 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/nvml/1128#-:-KQVpJj9HN4UQLIwtNw1:bnfp4nl)*
<!-- Sent from Reviewable.io -->
